frontend/micro-ui/web/micro-ui-internals/packages/modules/Masters/src/pages/employee/index.js

```diff
@@ -75,6 +76,7 @@ const App = ({ path }) => {
           <MastersBreadCrumb location={location} />
         </React.Fragment>
         <PrivateRoute path={`${path}/search-masters`} component={() => <SearchMasters parentRoute={path}/>} />
+        <PrivateRoute path={`${path}/create-masters`} component={() => <CreateMasters parentRoute={path}/>} />
```

Comment:
@prachirane-eGov  there is a route already for create org

Reply:
use the same , also this is not master

Reply:
mdms configured with the same

Reply:
Not updated the route, just renamed older component and added new component on same route

Reply:
ok